config/crd/bases/addons.cluster.x-k8s.io_helmchartproxies.yaml

```diff
@@ -88,11 +88,13 @@ spec:
                           items:
                             type: string
                           type: array
+                          x-kubernetes-list-type: atomic
```

Comment:
Why is dependabot messing with this CRD yaml? That seems out of scope for how it's configured.